src/script/cpp_api/s_base.cpp

```diff
@@ -106,7 +110,10 @@ ScriptApiBase::ScriptApiBase()
 	lua_newtable(m_luastack);
 	lua_setglobal(m_luastack, "core");
 
-	lua_pushstring(m_luastack, DIR_DELIM);
+	if (m_type == ScriptingType::Client)
+		lua_pushstring(m_luastack, "/");
```

Comment:
no, in CSM filesystem is virtual, there is no real path you only have a VFS and / is the delimiter